uber/sep_commands.py

```diff
@@ -14,19 +14,10 @@ def print_config():
 @entry_point
 def insert_admin():
     with Session() as session:
```

Comment:
ok so you're saying make the method be SessionMixin::insert_test_admin() instead of a standalone function, and make the sep command look like:

```python
@entry_point
def insert_admin():
    with Session() as session:
        session.insert_test_admin()
```

and the web call look like

```
session.insert_test_admin()
```